packages/utils/tsconfig.json

```diff
@@ -1,8 +1,5 @@
 {
   "extends": "@tsconfig/node18/tsconfig.json",
-  "ts-node": {
-    "files": true, // Load files, include and exclude from tsconfig.json on startup. This may avoid certain typechecking failures.
-  },
```

Comment:
This was added thinking it was required to compile packages/utils w/ references to the `packages/api`, but it turns out it's not needed. Removing it because it slows down the execution of scripts a lot.